src/main/scala/ctrl.scala

```diff
@@ -57,8 +57,6 @@ class CtrlModule(val W: Int, val S: Int)(implicit val p: Parameters) extends Mod
     val buffer_out  = Bits(OUTPUT,width=W)
   }
 
-  require(coreParams.dcacheReqTagBits >= 7)
```

Comment:
I agree that in its current form rocket-chip's tag bits will always be larger than the needed 5 bits.